.rubocop_todo.yml

```diff
@@ -0,0 +1,138 @@
+# Offense count: 9
```

Comment:
this looks a bad to have that todo file and especially when bunch of cops can be auto corrected like tabs one.

Reply:
merged into the .rubocop.yml file and fixed the **Layout/Tab** cop